README.md

```diff
@@ -21,83 +21,42 @@ The data is loaded from the `/data` directory.
 Each successful build will result in an image being pushed to Docker Hub. They are available at [nhsuk/profiles-db](https://hub.docker.com/r/nhsuk/profiles-db/). Pull Requests created via GitHub result in an image tagged as `pr-{PR#}` e.g PR #9 would result in an image tag of `pr-9`.
 When PRs are merged into the main branch i.e. `master` images are tagged with both `master` and `latest`.
 
-## Example JSON
-
-```
-{
-  "_id": 41272,
-    "choicesId": 41272,
-    "syndicationId": 14533,
-    "name": "A Kumar",
-    "odsCode": "H85056",
-    "address": {
-      "addressLines": [
-        "143-145 Balham Hill",
-      "Balham",
-      "London",
-      "London"
-      ],
-      "postcode": "SW12 9DL"
-    },
-    "location": {
-      "type": "Point",
-      "latitude": 51.4488334655762,
-      "longitude": -0.147822931408882
-    },
-    "contact": {
-      "telephone": "020 86731776",
-      "fax": "020 86739955"
-    },
-    "openingTimes": {
-      "reception": {
-        "sunday": [],
-        "monday": [
-        {
-          "opens": "08:00",
-          "closes": "13:00"
-        },
-        {
-          "opens": "15:00",
-          "closes": "19:00"
-        }
-        ],
-        "tuesday": [
-        {
-          "opens": "08:00",
-          "closes": "13:00"
-        },
-        {
-          "opens": "15:00",
-          "closes": "19:00"
-        }
-        ],
-        ...
-      },
-      "surgery": {
-        "sunday": [],
-        "monday": [
-        {
-          "opens": "08:00",
-          "closes": "13:00"
-        },
-        {
-          "opens": "15:00",
-          "closes": "19:00"
-        }
-        ],
-        "tuesday": [
-        {
-          "opens": "08:00",
-          "closes": "13:00"
-        },
-        {
-          "opens": "15:00",
-          "closes": "19:00"
-        }
-        ],
-      ...
-      }
-    }
-}
-```
+## Data structure
+An example of the structure of the data stored in MongoDB can be found in the [Sample GP Data](sample-gp-data.json)
+
+The more complex sub objects are described below
+
+### Address
+
+The `address` field consists of an array of `addressLines` and a `postcode`. The field is always present and there will always be a `postcode` and at least one `addressLine`.
+
+### Location
+
+The `location` field consists of a `type` field that is always `Point`, and `latitude` and `longitude` fields that are numbers. These fields are always present.
```

Comment:
Might be worth stating `location` is a [GeoJSON Point](http://geojson.org/geojson-spec.html#point). There is an example in that doc too, probably overkill to add the link in here